From: Michael Biebl <biebl@debian.org>
Date: Sun, 29 Mar 2015 22:58:43 +0200
Subject: systemd: Don't enable NetworkManager-wait-online.service statically

Instead we are going to hook up the service in network-online.target
dynamically via

[Install]
WantedBy=network-online.target

This way the user can easily disable the service.
---
 data/Makefile.am | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/data/Makefile.am b/data/Makefile.am
index 4ab4106..bbe6b32 100644
--- a/data/Makefile.am
+++ b/data/Makefile.am
@@ -18,10 +18,6 @@ endif
 
 NetworkManager-dispatcher.service: NetworkManager-dispatcher.service.in
 	$(edit) $< >$@
-
-install-exec-local:
-	install -d $(DESTDIR)$(systemdsystemunitdir)/network-online.target.wants
-	ln -sf $(systemdsystemunitdir)/NetworkManager-wait-online.service $(DESTDIR)$(systemdsystemunitdir)/network-online.target.wants
 endif
 
 examplesdir = $(docdir)/examples
